CAIPs/caip-350.md

```diff
@@ -0,0 +1,65 @@
+- Succinctness: Text formats necessarily have to incur encoding overhead compared to binary ones, causing relative informational inefficiency.
+
+## Specification
+The purpose of each namespace's profile is to specify deterministic and unambiguous conversions between the following format-pairs:
+- Customary address text formats of the ecosystem (which may also be described in its [CAIP-10] profile) and the Interoperable Names text representation defined in [ERC-7930]
+- Customary chain reference text formats of the ecosystem (which may also be described in [CAIP-2]) and the text representation defined in [ERC-7930]
+- [ERC-7930]'s Interoperable Address and [ERC-7930]'s Interoperable Name of the ecosystem's chain references
+- [ERC-7930]'s Interoperable Address and [ERC-7930]'s Interoperable Name of the ecosystem's addresses *for all address formats of said ecosystem*
+
+And must also define the binary id of the namespace itself, akin to [ERC-7930]'s `ChainType`
+
+For this, every namespace MUST define specify all of the above in a CAIP-350 profile to maximize interoperability and review. A [template for these profiles](https://github.com/ChainAgnostic/namespaces/blob/main/_template/caip350.md) is defined in the Namespaces registry.
+
+## Rationale
+The main alternative to this standard would've been to define all formats & conversions in [ERC-7930], turning it into a Living ERC, which was not ideal since:
+- Living standards are not a usual thing in the Ethereum ecosystem: the only other Living ERC is EIP-1
+- It would be desireable to finalize the definition of the format itself while allowing the specification of every chain's serialization to be defined as needed.
+- It would have placed all the editorial (and presumably most of the specification) work on the [ERC-7930] authors, while the CASA paradigm allows better distribution of work by leaving the definition of the profile for every chain namespace to the parties of its ecosystem interested in using Interoperable Addresses.
+
+## Test Cases
+Not applicable since this does not define a normative interface, instead being a meta-specification of normative interfaces.
+
+## Security Considerations
+It is possible that a CAIP-350 profile for a given chain namespace is not able to guarantee requirements of canonicity which are a desired property of Interoperable Addresses due to characteristics of the chain namespace, and systems relying on that canonicity may exhibit unexpected behavior as a result. When this is the case, it should be noted in the `Extra Considerations` section.
+
+## Backwards Compatibility
+This standard actively seeks to be exhaustive in its backwards compatibility with CAIP-2 and CAIP-10.
```

Comment:
I'm still not clear on how this works with Completeness, and whether a CAIP-350 implementation is allowed to produce or required to accept a CAIP-2 or CAIP-10 in cases where these truncate the information preserved in binary or in the novel format!